tests/csrf_tests/tests.py

```diff
@@ -353,6 +353,10 @@ def test_https_malformed_referer(self):
         req.META['HTTP_REFERER'] = 'https://'
         response = mw.process_view(req, post_form_view, (), {})
         self.assertContains(response, malformed_referer_msg, status_code=403)
+        # Invalid URL.
```

Comment:
It might be helpful to explain: "Invalid - urlparse() raises ValueError", or following the other examples:
```
>>> urlparse('https://[')
ValueError: Invalid IPv6 URL
```

Reply:
I had that example. I'll drop it in 👍